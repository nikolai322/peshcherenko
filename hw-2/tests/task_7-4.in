1qwe2we3
