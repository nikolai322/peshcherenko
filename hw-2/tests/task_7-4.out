1qwe2we3
aqwebwec
